config/options.go

```diff
@@ -561,6 +564,11 @@
 			return fmt.Errorf("config: bad forward-auth-url %s : %w", o.ForwardAuthURLString, err)
 		}
 		o.ForwardAuthURL = u
+		switch o.ForwardAuthType {
+		case ForwardingProxyNginx, ForwardingProxyTraefik:
+		default:
+			return fmt.Errorf("config: %s is required when forward-auth-url is set, supported types: %v\", o.ForwardAuthType, ForwardingProxyTypes", o.ForwardAuthType, ForwardingProxyTypes)
```

Comment:
err, I think you used my comment literally. This looks wrong and the line is way too long.

Reply:
Fixed